lib/features/campaigns/screens/statistics_screen.dart

```diff
@@ -1,20 +1,286 @@
+        color: theme.colorScheme.surfaceDim,
+        child: Column(
+          children: [
+            _getBadgeBox(statistics, context, theme),
+            SizedBox(height: 12),
+            _getCategoryBox(
+              stats: statistics.houseStats,
+              theme: theme,
+              title: t.campaigns.statistic.recorded_doors,
+            ),
+            SizedBox(height: 12),
+            _getCategoryBox(
+              stats: statistics.posterStats,
+              theme: theme,
+              title: t.campaigns.statistic.recorded_posters,
+              subTitle: t.campaigns.statistic.including_damaged_or_taken_down,
+            ),
+            SizedBox(height: 12),
+            _getCategoryBox(
+              stats: statistics.flyerStats,
+              theme: theme,
+              title: t.campaigns.statistic.recorded_flyer,
+            ),
+            Container(
+              padding: EdgeInsets.all(16),
+              child: Align(
+                alignment: Alignment.centerLeft,
+                child: Text(
+                  '${t.campaigns.statistic.as_at}: ${lastUpdateTime.getAsLocalDateTimeString()} (${t.campaigns.statistic.update_info})',
+                  style: theme.textTheme.labelMedium!.apply(color: ThemeColors.textDisabled),
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Widget _getBadgeBox(CampaignStatistics statistics, BuildContext context, ThemeData theme) {
+    var mediaQuery = MediaQuery.of(context);
+    return Container(
+      padding: EdgeInsets.all(16),
+      width: mediaQuery.size.width,
+      decoration: BoxDecoration(
+        color: ThemeColors.background,
+        borderRadius: BorderRadius.circular(19),
+        boxShadow: [
+          BoxShadow(color: ThemeColors.textDark.withAlpha(10), offset: Offset(2, 4)),
+        ],
+      ),
+      child: Column(
+        children: [
+          Align(
+            alignment: Alignment.centerLeft,
+            child: Text(
+              t.campaigns.statistic.my_badges,
+              style: theme.textTheme.titleMedium,
+            ),
+          ),
+          Align(
+            alignment: Alignment.centerLeft,
+            child: Text(
+              t.campaigns.statistic.my_badges_campaign_subtitle,
+              style: theme.textTheme.labelSmall,
+            ),
+          ),
+          ..._getBadges(statistics, theme),
+        ],
+      ),
+    );
+  }
+
+  List<Widget> _getBadges(CampaignStatistics statistics, ThemeData theme) {
+    return [
+      _getBadgeRow(t.campaigns.statistic.recorded_doors, statistics.houseStats.own.toInt(), theme),
+      _getBadgeRow(t.campaigns.statistic.recorded_posters, statistics.posterStats.own.toInt(), theme),
+      _getBadgeRow(t.campaigns.statistic.recorded_flyer, statistics.flyerStats.own.toInt(), theme),
+    ];
+  }
+
+  Widget _getBadgeRow(String title, int ownCounter, ThemeData theme) {
+    // var rng = Random();
```

Comment:
[Here comes the sun](https://www.youtube.com/watch?v=dbEOEt-sPeE)
(but nonetheless you might want to delete this).